modules/tile-converter/src/i3s-converter/i3s-converter.js

```diff
@@ -87,12 +87,12 @@ export default class I3SConverter {
     }
 
     const options = await this._fetchPreloadOptions();
-    const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, options);
-
-    /* TODO/ib - get rid of confusing options warnings, move into options sub-object */
-    // const tilesetJson = await load(inputUrl, CesiumIonLoader, {
-    //   'cesium-ion': preloadOptions
-    // });
+    const sourceTilesetJson = await load(inputUrl, CesiumIonLoader, {
+      'cesium-ion': options['cesium-ion'],
+      fetch: {
```

Comment:
Can we just stop using options.headers in the caller and move to `options.fetch` instead of patching it here?

Reply:
Moved all options to fetch property